pygeoapi/util.py

```diff
@@ -49,16 +51,21 @@
 
 LOGGER = logging.getLogger(__name__)
 
+DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+
 TEMPLATES = '{}{}templates'.format(os.path.dirname(
     os.path.realpath(__file__)), os.sep)
 
+ALLOWED_UPLOAD_EXTENSIONS = {'csv', 'shp', 'json', 'geojson', 'gpkg'}
+UPLOAD_FOLDER = os.getenv('UPLOAD_FOLDER', '/tmp')
```

Comment:
Unlcear where the boundary is between a manager and the core API? (I think this would be necessary for file upload, but that's absent now.)

Reply:
The core API includes a "dummy" manager (which always forces synchronous execution).